lib/grpc/stub.ex

```diff
@@ -164,6 +164,7 @@ defmodule GRPC.Stub do
 
   * When using DNS or xDS targets, the connection layer periodically refreshes endpoints.
   """
+  @doc type: :client_connection
```

Comment:
Stub is client by definition.
Also, there are so few functions here I don't think there's much to gain in using types